lib/pyre/tensor/repacking.h

```diff
@@ -29,20 +29,20 @@ namespace pyre::tensor {
 
     template <int N, template <typename, std::size_t> class containerT>
     struct repacking_sum<
-        pyre::grid::Canonical<N, containerT>, pyre::grid::Canonical<N, containerT>> {
-        using packing_type = pyre::grid::Canonical<N, containerT>;
+        pyre::grid::Canonical<N, int, containerT>, pyre::grid::Canonical<N, int, containerT>> {
```

Comment:
Occurrences of `pyre::grid::Canonical<N, int, containerT>` can be edited to  use the api type alias `pyre::grid::canonical_t<N, int, containerT>` as @aivazis suggested and possibly simplified into `pyre::grid::canonical_t<N>`, therefore using the default types for the template parameters (int and std::array). @aivazis what do you think? Shall we do this in this PR? In case, I can implement this change.

Reply:
sounds like you think this a good idea. i'll take care of it